pandas/core/generic.py

```diff
@@ -4285,6 +4307,31 @@ def ftypes(self):
         """
         Return the ftypes (indication of sparse/dense and dtype)
```

Comment:
The docstring guide asks that the summary should fit in a single line. Could you rephrase it that way? If all information  could not fit in a single line you can then use an Extended Summary section.